src/SQLite.cs

```diff
@@ -3573,34 +3573,40 @@ private CompileResult CompileExpr (Expression expr, List<object> queryArgs)
 				}
 				else if (call.Method.Name == "StartsWith" && args.Length >= 1) {
 					var startsWithCmpOp = StringComparison.CurrentCulture;
+					string sDummyCondForParamTwo = string.Empty;
```

Comment:
Hello,
Your suggestion does not work when doing exactly like this, because the parameter is lost like this and the ignoring case option is lost like this.
I did a solution close to your suggestion, which just prevents that the parameter is added to queryargs but not to args.
```
List<object> notUsedQueryArgs = new List<object> ();
for (var i = 0; i<args.Length; i++) {
	if (i == 1 && (call.Method.Name == "StartsWith" || call.Method.Name == "EndsWith")) {
		args[i] = CompileExpr (call.Arguments[i], notUsedQueryArgs);
	}
	else {
		args[i] = CompileExpr (call.Arguments[i], queryArgs);
	}
}
```